LW_Equation/LinearEquation.cs

```diff
@@ -10,44 +10,120 @@ public class LinearEquation
     {
         List<float> coefficients;
         public int Size => coefficients.Count;
-        public LinearEquation(float b, float aN, params float[] coefficients)
+
+        public LinearEquation(params float[] coefficients)
         {
             this.coefficients = new List<float>();
-            this.coefficients.Add(aN);
-            this.coefficients.Add(b);
             this.coefficients.AddRange(coefficients);
         }
         public LinearEquation(List<float> coefficients)
         {
             this.coefficients = new List<float>();
             this.coefficients = coefficients;
         }
+        public LinearEquation(bool t, int size)
```

Comment:
Параметр t не используется, можно удалить и создать класс EquationSize или что-то похожее, и заменить тип size на EquationSize.
```suggestion
        public LinearEquation(EquationSize size)
```
Новый класс может быть таким:
```csharp
class EquationSize
{
  public int Size {get;set;}
  public EquationSize(int size)
  {
    Size = size;
  }
  public static implicit operator int(EquationSize size)
  {
    return size.Size;
  }
}
```